.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,23 @@
+name: build and test gradle project
+
+on:
+  push:
+    branches-ignore:
+    - master
+    - version-*
+    - dependabot*
```

Comment:
I don't think we need this patterns here. I believe we will have only the master branch